### PR TITLE
Added csv logger for allowed/denied requests

### PR DESCRIPTION
**Pull request by ps100000**
I added a logger which writes down allowed/denied requests in a CSV file.
I also added a settings dash for it.

### CODE REVIEW

**Comment by alexa-v2** [comment]
Looking good to me, but have to wait for @kar

File: app/src/flavor-adblocker/kotlin/adblocker/RequestLogger.kt [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+    private fun time() = Date().time.toString(10)
+}
+
+class LoggerDash(
+        val ctx: Context
+) : Dash(
+        "logger_settings",
+        icon = R.drawable.ic_tune,
+        text = "Logger settings",
```
Comment (kar) [inline app/src/flavor-adblocker/kotlin/adblocker/RequestLogger.kt]: Please use string resources

File: app/src/legacy/res/layout/view_logger_config.xml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center_horizontal"
+        android:orientation="vertical"
+        android:padding="8dp">
+
+        <!-- Status -->
```
Comment (kar) [inline app/src/legacy/res/layout/view_logger_config.xml]: Please drop irrelevant comments.

File: app/src/legacy/res/values/strings_logger.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="logger_permission">please grand storage permission first.</string>
```
Comment (kar) [inline app/src/legacy/res/values/strings_logger.xml]: Start sentence with uppercase.

**Comment by kar** [comment]
I have enabled storage permissions and restarted the app, but I can't find the file anywhere. What should I look for in the logcat?

**Comment by kar** [comment]
Works great.

**Pull request closed without merging**